Gemfile

```diff
@@ -44,6 +43,13 @@ group :development, :test do
   gem "quiet_assets", "~> 1.1"
   gem "rspec-rails", "~> 3.2.0"
   gem "rubocop", "~> 0.30"
+  gem "phantomjs", "~> 1.9.7"
+  gem "jasmine-core", "~> 2.0.0"
```

Comment:
It took a while to get a stable combination of gem versions to run `jasmine` with `phantomjs`.  `jasmine-core` is a dependency required by `jasmine`:
```ruby
    jasmine (2.0.3)
      jasmine-core (~> 2.0.0)
```
I'll remove the explicit require, and commit if everything runs ok.